db/migrate/20220617140542_move_become_method_from_options.rb

```diff
@@ -0,0 +1,29 @@
+class MoveBecomeMethodFromOptions < ActiveRecord::Migration[6.0]
+  class Authentication < ActiveRecord::Base
+    include ActiveRecord::IdRegions
+    self.inheritance_column = :_type_disabled
+
+    serialize :options
+  end
+
+  def up
+    say_with_time("Move Authentication become_method out of options text field") do
+      Authentication.in_my_region.find_each do |authentication|
```

Comment:
can we tack on a where clause?

```suggestion
      Authentication.in_my_region.where("options like '%become_method%'").find_each do |authentication|
```

Reply:
capital `LIKE`, but I agree.